src/ptf/mask.py

```diff
@@ -87,13 +98,11 @@ def __str__(self):
         assert self.valid
```

Comment:
Why do we have here this assert? In packet not match case, when we want to print mask into logs (expected/actual) it will raise AssertionError, thus loosing all the information about packet mismatch.

Reply:
You mean, something like this? https://github.com/p4lang/ptf/pull/165/commits/79aeeac3606fec364d31c2bf82cba269bb9a0b7a

I especially add "OK" & "INVALID" to the status. "VALID" & "INVALID" can be confusing... .